components/Ripple/src/MDCStatefulRippleView.m

```diff
@@ -144,15 +144,13 @@ - (void)setSelected:(BOOL)selected {
   _selected = selected;
   // Go into the selected state visually.
   if (selected) {
+    [self updateActiveRippleColor];
```

Comment:
i wouldn't make any changes here as this code is pretty fragile, why were these changes needed?

Reply:
I think its gonna be fine if we remove this change, since I added the getter change. This change here was based the original logic. It was using ripple color as the value to set activeLayer before.

Reply:
I can move the change there to a separate PR. Though it would affect the test, I think it worth to clean up the fragile dependent logic here.

Reply:
Removed in 95206a3b5329c4009ec198ef11921a73d8358534.